server/pbench/bin/pbench-base.sh

```diff
@@ -1,5 +1,8 @@
 #! /bin/bash
 
+# force UTC everywhere
+export TZ=UTC
```

Comment:
Did this change modify any gold files in and of itself?  If so, we should probably make this change as a separate commit.

Reply:
No, it was only affecting the results I was getting on my laptop (which is using local time).

Reply:
Probably worth making that a separate commit anyways.  That could save folks a lot of trouble and it would be good to have that exposed in the history.

Reply:
Split into a separate PR #910  which is a dependency for this PR. I'll rebase after #910 is merged.